docs/separation-standards/runway.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Runway
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Takeoff
+### Behind a preceding departing aircraft
+- Preceding departing aircraft;  
+    - has crossed the up-wind end of the runway-in-use; or  
+    - has commenced a turn; or  
+    - is airborne and has reached a point at least 1800m (6000ft) ahead of the following aircraft provided the runway is longer than 1800m (6000ft); and  
```

Comment:
600m standard for GA stuff?

Reply:
It wasn't in the moodle, I assumed because of all of the complex MTOW conditions associated with it. But I can add it in